bin/cli.js

```diff
@@ -94,9 +102,19 @@ async function bundle(main, command) {
   const bundler = new Bundler(main, command);
 
   if (command.name() === 'serve') {
-    const server = await bundler.serve(command.port || 1234);
+    const server = await bundler.serve(command.port, command.host);
+
     if (command.open) {
-      require('opn')(`http://localhost:${server.address().port}`);
+      const open = () => {
+        require('opn')(
+          `http://${command.host || 'localhost'}:${server.address().port}`
+        );
+      };
```

Comment:
The below fixes a bug. Timing wasn't an issue when using `localhost` but `listening` appears to be slower (at least on Windows) when providing a host. This results in an attempted `server.address().port` call before `listening`. Simply defer if not `listening` yet.

Reply:
You are setting the default host here, but not the default port ...